src/autopas/tuning/AutoTuner.cpp

```diff
@@ -280,6 +283,14 @@ void AutoTuner::addMeasurement(long sample, bool neighborListRebuilt) {
 void AutoTuner::bumpIterationCounters() {
   ++_iteration;
   ++_iterationsSinceTuning;
+  if (inTuningPhase()) {
+    ++_iterationsInLastTuningPhase;
+    if (_iterationsInLastTuningPhase > _tuningInterval) {
+      throw autopas::utils::ExceptionHandler::AutoPasException(
+          "Tuning needs more iterations than the tuning interval. Please use a higher value for the tuning interval.");
```

Comment:
```suggestion
      utils::ExceptionHandler::exception(
          "Tuning needs more iterations than the tuning interval ({}). Please use a higher value for the tuning interval.", _tuningInterval);
```
The wrapper function provides formatting.